cql/CollateManagementData.cql

```diff
@@ -259,7 +259,7 @@ define MostRecentCytologyReportDate:
     MostRecentCytologyReport.date
   else
     null
-
+    
```

Comment:
This PR probably doesn't need to touch this file now, but not a huge deal, just could be helpful to remove for git tracking purposes.

Reply:
removed whitespace